dfkernel/dataflow.py

```diff
@@ -427,9 +427,10 @@ def add_links(self, output_tags):
         new_tags = set()
         for (cell_id, tag_list) in output_tags.items():
             for tag in tag_list:
-                # print("OUTER ADD_LINKS:", cell_id, tag)
-                self.add_link(tag, cell_id, make_current=False)
-                new_tags.add(tag)
+                if tag:
```

Comment:
Why is this necessary? You're calling output_tags.items(), you're getting back things that are None?

If so why are there None/NA values in the output_tags and why do we allow it that? It seems weird that we'd be catching this here.

Reply:
Condition is removed and will be handled in dfnotebook extension code